MANIFEST.in

```diff
@@ -5,17 +5,17 @@ global-exclude .git*/**
 global-exclude .travis*
 global-exclude .travis/**
 global-exclude codebuild/**
-global-exclude .cbmc-batch/**
+global-exclude crt/*/verification/**
 global-exclude crt/*/docs/**
 global-exclude docker-images/**
 prune crt/**/AWSCRTAndroidTestRunner
 prune crt/aws-c-auth/tests/aws-sig-v4-test-suite
 prune crt/aws-c-auth/tests/fuzz/corpus
-prune crt/s2n/tests/fuzz/corpus
-prune crt/s2n/tests/ctverif
-prune crt/s2n/tests/pems
-prune crt/s2n/tests/saw
-prune crt/s2n/tests/sidetrail
```

Comment:
I guess s2n doesn't have these anymore because of aws-lc?

Reply:
I just pruned the entire test suite, it's not necessary to reproduce the wheel or build from source

Reply:
Oh, I see now, it just nukes the entire "tests" directory.  My mistake.